desktop-src/Controls/themesfileformat-overview.md

```diff
@@ -720,9 +718,21 @@ Supported file types include the following:
 | Desktop icon | .ico                                |
 | Brand logo   | .png                                |
 
+If you place these asset files in the root of the .cab, you can reference most of them in your .theme file directly. For example, if you have a file called `Alert.wav` in the root of your .cab, you can use it in your sound scheme:
```

Comment:
```suggestion
If you place these asset files in the root of the .cab, you can reference them in your .theme file directly. For example, a file called `Alert.wav` in the root of your .cab could be used in your sound scheme as follows:
```

why "most of them"?

Reply:
I am not sure why I initially caveated this statement, but looking now I'm worried that there are yet other file types that must live in other directories.

I've revised the text to be more precise, and I've specifically added "like sounds" to indicate what _definitely does_ work 😄.

That's why I wanted the theming experts to check this in 2020.